lib/core/configurator.c

```diff
@@ -554,6 +554,10 @@ static int assert_is_extension(h2o_configurator_command_t *cmd, yoml_t *node)
         h2o_configurator_errprintf(cmd, node, "given extension \"%s\" does not start with a \".\"", node->data.scalar);
         return -1;
     }
+    if (strlen(node->data.scalar) == 1) {
```

Comment:
This is definitely a nitpick, but how about changing to `node->data.scalar[1] == '\0'`?